clusterloader2/testing/list/README.md

```diff
@@ -0,0 +1,16 @@
+## List load test
+
+List load test perform the following steps:
+- Create configmaps and secrets in a set of namespaces.
+  - The namespaces used here are managed by CL2.
+  - The number of namespaces can be specified using `CL2_LIST_NAMESPACE_NUMBER`.
```

Comment:
This has to be changed to reflect [this](https://github.com/kubernetes/perf-tests/pull/2302#discussion_r1370579581)

Reply:
Thanks for catching this.
Updated